src/components/PageBlocks/Wrap/Card.vue

```diff
@@ -5,6 +5,11 @@
       class="h-100 border-0 shadow"
       :class="blockClass"
     >
+      <font-awesome-icon
+        v-if="block.options.refreshRate >= 5"
```

Comment:
This will make the icon be hidden on certain blocks, i suggest you do what we talked about and just add this to the header title. If there is no title, we just wont show the icon